database_schemas/cassandra/benchflow.cql

```diff
@@ -394,7 +400,18 @@ CREATE TABLE exp_faban_ops_custom_stats (
   faban_stat_mode list<double>,
   faban_stat_mode_freq bigint,
   faban_stat_sd double,
-  PRIMARY KEY (experiment_id, faban_driver_host, faban_driver_name, faban_op_name, faban_op_stat_name)
+  PRIMARY KEY (experiment_id, faban_driver_host, faban_driver_name, faban_stat_name)
+);
+
+CREATE TABLE exp_faban_ops_custom_stats_composite (
```

Comment:
@VincenzoFerme Did I get that backwards? In composite I save the computations done on data that is already stats, so all I can do is min/max, since I don't know what they are. In the other table I save stats computed on some metric, so all of our defined ones.